motion/mpu6050.h

```diff
@@ -100,6 +100,109 @@ class MPU6050 : public I2CDevice, Looper, StateMachine {
 
   MPU6050() : I2CDevice(0x68) {}
 
+#ifdef USE_TEENSY4 //use Wire.h and ASYNC to read MPU6050
```

Comment:
I think we should replace this with something like "USE_ASYNC_MOTION" or something and then define that in the teensy4 config file instead.

Also, there should be a space after //